File: triangulo/triangulo.py
lado1 = int (input("Digite o primeiro lado: "))
lado2 = int (input("Digite o segundo lado: "))
lado3 = int (input("Digite o terceiro lado: "))


if lado1 + lado2 < lado3 or lado2 + lado3 < lado1 or lado1 + lado3 < lado2:
    print('Não é um triângulo')
elif lado1 == lado2 and lado1 == lado3:
    print('Equilátero')
elif lado1 == lado2 or lado1 == lado3 or lado3 == lado2:
    print('Isósceles')
else:
    print('Escaleno')

File: triangulo/triangulo.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("Digite o primeiro lado: ");
        int lado1 = input.nextInt();
        System.out.println("Digite o segundo lado: ");
        int lado2 = input.nextInt();
        System.out.println("Digite o terceiro lado: ");
        int lado3 = input.nextInt();
        input.close();

        if (lado1 + lado2 < lado3 || lado2 + lado3 < lado1 || lado1 + lado3 < lado2) {
            System.out.println("Não é um triângulo\n");
        }
        
        else if(lado1 == lado2 && lado1 == lado3) {
            System.out.println("Equilátero\n");
        }
    
        else if(lado1 == lado2 || lado1 == lado3 || lado3 == lado2) {
            System.out.println("Isósceles\n");
        }
    
        else {
            System.out.println("Escaleno\n");
        }
        
    }
}